### PR TITLE
Set bitcoind log file under /var/log

### LINKED ISSUE #194: Bug: bitcoind log file should be in /var/log, not datadir
**Have you read the documentation?**
Yes. (If no, please go check if this is a documented property.)

**Describe the bug**
bitcoind log file is in it's home (datadir). Log files belong to /var/log

**To Reproduce**
1. Install package: `bitcoind`
2. `ls /var/log/bitcoin-mainnet`
3. `ls /var/lib/bitcoin-mainnet`
4. See `debug.log` in the latter

**Expected behavior**
`debug.log` is in `/var/log/bitcoin-mainnet`

**System**
OS, if anything else than Debian Buster:
Installed packages from this repository and their versions: `bitcoind` 0.21.1
Is apparmor enabled? Irrelevant

### PR DESCRIPTION
**Pull request by HollowMan6**
Resolve #194

Signed-off-by: Hollow Man <hollowman@opensuse.org>

### CODE REVIEW

File: pkg_specs/bitcoin-@variant.sps [hunk 1]
```diff
@@ -49,6 +49,14 @@ The full path to the directory which will contain timechain data (blocks and cha
 Important: you need around 10GB of free space!
 """
 
+[config."bitcoin.conf".ivars.debuglogfile]
+type = "path"
+file_type = "regular"
+create = { mode = 755, owner = "$service", group = "$service" }
```
Comment (Kixunil) [inline pkg_specs/bitcoin-@variant.sps:55]: Better let bitcoind create it, we don't need to. (IOW, delete this line; note that mode would've been incorrect too)
However the parent should be created, ideally by systemd - add:

```
LogsDirectory=bitcoin-{variant}
```

to `extra_service_config`

File: pkg_specs/bitcoin-@variant.sps [hunk 2]
```diff
@@ -49,6 +49,14 @@ The full path to the directory which will contain timechain data (blocks and cha
...
+default = "/var/log/bitcoin-{variant}/debug.log"
+priority = "medium"
+summary = "File containing the bitcoind debug logs"
```
Comment (Kixunil) [inline pkg_specs/bitcoin-@variant.sps:58]: Would be nice to mention the network, e.g. "File containing the bitcoind debug logs {variant}"

**Review by Kixunil — CHANGES_REQUESTED** [review]
Thanks! Just missing some minor details.

**Comment by HollowMan6** [comment]
@Kixunil Done! Thanks for reviewing.

**Comment by Kixunil** [comment]
WTF, I'd swear I've merged this. Let's ignore the failed test, it's not relevant to this.

**Comment by Kixunil** [comment]
Oh, wait, I know why I didn't mere. The locations are not synchronized. It Either has to be a hidden var to prevent changing or definitions must read the configuration.

**Comment by HollowMan6** [comment]
Oh okay, just make the definitions read the configuration as I think that would be better to prevent the log location from changing.

**Pull request merged**